ubuntudesign/www.ubuntu.com
# add Ubuntu Advantage shop to global nav

## Done

* updated global nav to:
 * remove Discourse
 * replace the link to shop to buy.ubuntu.com
 * add new merchandise link in More
 * remove Apps link as that site is no longer supported
* removed accessibility-aide with list of things to do on forms as they are out of date and not used
* replaces urls for shop to buy in a few places
* removed old header.html file

## QA

* see that the new nav is loading and correct
* see that there are no links to the old shop

## Issue / Card

[trello card](https://trello.com/c/h2ZF0lon)
